dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/DotRestHighLevelClientProvider.java

```diff
@@ -104,6 +113,23 @@ private BasicCredentialsProvider getCredentialsProvider() throws IOException, Ge
         return credentialsProvider;
     }
 
+    private ClientConfiguration getReactiveClientBuilder(){
+        ClientConfiguration clientConfiguration = ClientConfiguration.builder()
+                .connectedTo(Config.getStringProperty("ES_HOSTNAME", "127.0.0.1") + ":" + Config.getIntProperty("ES_PORT", 9200))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not hard code the IP address ](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5113278)